raiden-ts/src/transport/epics/helpers.ts

```diff
@@ -148,15 +149,29 @@ export function waitMemberAndSend$<C extends { msgtype: string; body: string }>(
       ),
       pluck('rtc', address),
     ),
+    // if available and Capabilities.TO_DEVICE enabled on both ends, use ToDevice messages
+    latest$.pipe(
+      pluck('presences', address),
+      withLatestFrom(config$),
+      filter(
+        ([presence, { caps }]) =>
+          !!caps?.[Capabilities.TO_DEVICE] &&
+          !!presence?.payload?.available &&
+          !!presence.payload.caps?.[Capabilities.TO_DEVICE],
+      ),
+      pluck('0', 'payload', 'userId'),
+    ),
     waitMember$(matrix, address, { latest$ }),
   ).pipe(
     take(1), // use first room/user which meets all requirements/filters above
     mergeMap((via) =>
-      defer<Promise<void>>(
+      defer<Promise<unknown>>(
         async () =>
-          typeof via === 'string'
-            ? matrix.sendEvent(via, type, content, '') // via room
-            : via.send(content.body), // via RTC channel
+          typeof via !== 'string'
+            ? via.send(content.body) // via RTC channel
+            : via.startsWith('@')
```

Comment:
Can you use if statements here? I find those nested ternary operators quite hard to read.

Reply:
Sure, here it's possible, I will do, but it's better to get used to them, unfortunately they're quite hard to avoid while preserving type-safety on `const`s on TypeScript, and alternatives are not always as reliable.

Reply:
This was already fixed on latest commit.